ros_ws/launch/gazebo_car-autonomous.launch

```diff
@@ -10,21 +9,22 @@
     <arg name="debug" default="false"/>
     <arg name="verbose" default="true"/>
     <arg name="use_gpu" default="true"/>
+    <arg name="visualize_lidar" default="false"/>
 
-    <include file="$(find racer_world)/launch/racer_gazebo_rviz.launch">
+    <include file="$(find racer_world)/launch/racer_gazebo.launch">
         <arg name="world" value="$(arg world)"/>
         <arg name="paused" value="$(arg paused)"/>
         <arg name="use_sim_time" value="$(arg use_sim_time)"/>
         <arg name="gui" value="$(arg gui)"/>
         <arg name="headless" value="$(arg headless)"/>
         <arg name="debug" value="$(arg debug)"/>
         <arg name="verbose" value="$(arg verbose)"/>
-        <arg name="use_gpu" value="$(arg use_gpu)"/>
```

Comment:
Why did you remove the use_gpu agument? 
You did not do that in gazebo_car-teleop.launch

Reply:
This was unintentional and is now fixed.